docs/synthetics/status-pages.md

```diff
@@ -11,7 +15,7 @@ You can create multiple status pages under your account. Navigate to the `Synthe
 
 The following fields are required to create a status page:
 
-* **Monitors** - List of monitors to be included in the status page. All the active monitors from Apps for which the account has OWNER or ADMIN role can be included in the status page. A monitor can be included in multiple status pages.
+* **Monitors** - List of monitors to be included in the status page. All the active monitors from Apps for which you have OWNER or ADMIN role can be included in the status page. A monitor can be part of multiple status pages.
```

Comment:
A monitor can be part of multiple status pages. ==> The same monitor can be included in multiple status pages.